src/main/pages/workspace-admin/recipes.adoc

```diff
@@ -100,34 +143,28 @@ networks:
   driver: bridge
 ----
 
-Che does not yet currently support compose networks. We do, however, support the use of aliases through `links` command.
-
 *Hostname*
 
-Hostname is not supported and the machine’s name will be used for the hostname. User can use `links` aliases syntax to add additional hostname(s) to a machine.
+The hostname is not supported.  The machine’s name is used for the hostname. You can use `links` aliases syntax to add additional hostnames to a machine.
 
-*Ports*
+*Binding ports*
 
-Binding ports to the host system is not supported. This is done to ensure that each container does not used already assigned host ports because many developers could be running the same compose file on the same Che server host at the same time. Users can add link:servers.html[servers] to machines though.
+Binding ports to the host system is not supported.  You must ensure that each container does not use already assigned host ports.  Many developers can be running the same Compose file on the same Che server host at the same time. You can add link:servers.html[servers] to the machines.
 
-*Privileged*
+*Environment file*
 
-To secure the underlying host system `privileged` compose command is not supported.
+The `env_file` compose command is not supported. Environment variables can be manually entered in the Compose file or machine configuration. See link:env-variables.html[Environment variables] for more information.
```

Comment:
Should "compose" be capitalized in "compose command" here?

Reply:
I am looking into this..

Reply:
It's not a command -- it's an option. See https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option. So yes, Compose should be capitalized.